.teamcity/settings.kts

```diff
@@ -891,11 +899,16 @@ object RunCanaryE2eTests : BuildType({
 				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
 
 				# Run the test
-				./run.sh -R -a %E2E_WORKERS% -C -s "mobile,desktop" -u "${'$'}{URL%/}"
+				export BROWSERSIZE="desktop"
+				export BROWSERLOCALE="en"
+				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
+				export TEST_FILES=${'$'}(join ',' ${'$'}(ls -1 specs*/**/*spec.js))
+
+			yarn magellan --config=magellan.json --max_workers=%E2E_WORKERS% --suiteTag=parallel --local_browser=chrome --test=${'$'}{TEST_FILES}
```

Comment:
This test uses `yarn magellan` directly instead of the wrapper `run.sh`. The reason is `run.sh` doesn't have a combination of args that allows us to specify the list of files _and_ not run on CircleCI.

Changing `run.sh` was a bit too risky, as it can potentially affect existing CircleCI runs. Instead, having the direct call here makes it easier to iterate on and simpler to understand what is going on.